tests/sample_agents/test_parrot_agent.py

```diff
@@ -27,22 +27,22 @@ def agent() -> ParrotAgent:
 
 
 @pytest.fixture
-def dm(user: User, agent: ParrotAgent) -> DialogueManager:
-    """Dialogue manager fixture."""
-    dm = DialogueManager(agent, user, Platform(), save_dialogue_history=False)
-    return dm
+def dc(user: User, agent: ParrotAgent) -> DialogueConnector:
+    """Dialogue connector fixture."""
+    dc = DialogueConnector(agent, user, Platform(), save_dialogue_history=False)
```

Comment:
Nit: use `connector` instead of `dc` for improved readability?

Reply:
Done.